Bottom: 4c315f0029427de2b159a3dcb9ceb27e69d6d0ae
Top:    351d96f57ed7d86ccde0d5aa930ba85ff8ba50c6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-05 00:08:36 +0100

Refresh of support-holes-in-input-polygon

---

diff --git a/src/create.c b/src/create.c
index f419d2a..c73b399 100644
--- a/src/create.c
+++ b/src/create.c
@@ -645,8 +645,6 @@ CreateNewHoleInPolygon (PolygonType *Polygon)
 {
   int *holeindex = GetHoleIndexMemoryInPolygon (Polygon);
   *holeindex = Polygon->PointN;
-  printf ("Creating hole in polygon %i, Hole %i, PointIndex %i\n",
-          Polygon->ID, Polygon->HoleIndexN, *holeindex);
   return Polygon;
 }
